bindings/python/core_types/include/py_core_types/py_converter.h

```diff
@@ -35,14 +36,21 @@ inline py::array_t<typename Sequence::value_type> toPyArray(Sequence&& seq, cons
     seq_ptr.release();
 
     py::dtype dt = py::dtype::of<typename Sequence::value_type>();
-    if(!dtype.empty()) {
-        //it looks like datetime64 is ignored by pybind11 now, so we need to use WA
-        //this is for future use
+    if (!dtype.empty())
+    {
+        // it looks like datetime64 is ignored by pybind11 now, so we need to use WA
+        // this is for future use
         dt = py::dtype(dtype);
     }
 
     py::array::ShapeContainer arrayShape = {size};
-    if(!shape->empty()) arrayShape = std::move(shape);
+    if (!shape->empty())
+        arrayShape = std::move(shape);
+
+    if (strides->empty())
+    {
+        return py::array(dt, arrayShape, data, capsule);
+    }
```

Comment:
Just nitpicking. I would say that it's better to have the code consistent: either `{}` everywhere for single-line statements, or nowhere (two lines above it's without).
```suggestion
        return py::array(dt, arrayShape, data, capsule);
```